care/utils/serializer/external_id_field.py

```diff
@@ -0,0 +1,39 @@
+import uuid
+
+from django.core.exceptions import ObjectDoesNotExist
+from rest_framework import serializers
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5498155)